src/functionalTest/resources/application-integration.yml

```diff
@@ -0,0 +1,83 @@
+spring:
+  mail:
+    from: no-reply@reform.hmcts.net
+    to:  sushant.choudhari@hmcts.net,abhijit.diwan@hmcts.net,mallikarjun.puttana@hmcts.net
+    subject:  JRD - Exception in processing -
+    host: mta.reform.hmcts.net
+    port: 25
+    enabled: true
+    properties:
+      mail-smtp:
+        starttls:
+          enable: false
+        ssl:
+          trust: "*"
+  flyway:
+    locations: classpath:db/testmigration
+    enabled: true
+  batch:
+    initialize-schema: always
+    # set batch false in integration test and run batch manually with batch runner to set dynamic test property values
+    # eg. dynamic properties ${parent.file.name}
+    job:
+      enabled: false
+
+#Local integration testing these values should be actual
+#ACCOUNT_NAME: ${ACCOUNT_NAME}
+#ACCOUNT_KEY: ${ACCOUNT_KEY}
+
+start-route: direct:LRD
```

Comment:
can camel related properties be moved to another properties file ?

Reply:
This is integartoin test properties..hence that is fine not source properties